### PR TITLE
Add documentation for the `getType()` builtin method

### LINKED ISSUE #733: Add documentation for the `getType()` builtin method
### Issue To Be Solved

Add documentation for the `getType()` builtin method.

Could add it under https://docs.onflow.org/cadence/language/run-time-types, similar to the `isInstance` method.

### PR DESCRIPTION
**Pull request by SupunS**
Closes #733

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

**Review by joshuahannan — APPROVED** [review]
Looks good. Thank you for adding this!

File: docs/language/run-time-types.md [hunk 1]
```diff
@@ -31,9 +31,31 @@ Type<Int>() == Type<Int>()
 
 Type<Int>() != Type<String>()
 ```
+### Getting the Type from a Value
+The method `fun getType(): Type` can be used to get the runtime type of a value.
 
+```cadence
+let something: String = "hello"
+
+var type :Type = something.getType()  // `type` is `Type<String>()`
```
Comment (turbolent) [inline docs/language/run-time-types.md:40]:
```suggestion
let something = "hello"

let type: Type = something.getType()
// `type` is `Type<String>()`
```

File: docs/language/run-time-types.md [hunk 2]
```diff
@@ -31,9 +31,31 @@ Type<Int>() == Type<Int>()
...
+
+```cadence
+// Declare a variable named `something` that has the *static* type `AnyResource`
+// and has a resource of type `Collectible`
+//
+let something: @AnyResource <- create Collectible()
+
+// The resource's concrete run-time type is `Collectible`
+//
+var type :Type = something.getType()  // `type` is `Type<@Collectible>()`
```
Comment (turbolent) [inline docs/language/run-time-types.md:53]:
```suggestion
let type: Type = something.getType()
// `type` is `Type<@Collectible>()`
```

File: docs/language/run-time-types.md [hunk 3]
```diff
@@ -31,9 +31,31 @@ Type<Int>() == Type<Int>()
...
+### Getting the Type from a Value
+The method `fun getType(): Type` can be used to get the runtime type of a value.
 
+```cadence
+let something: String = "hello"
+
+var type :Type = something.getType()  // `type` is `Type<String>()`
+```
+
+This method returns the **concrete run-time type** of the object, **not** the static type.
```
Comment (turbolent) [inline docs/language/run-time-types.md:43]: 👍

**Review by turbolent — APPROVED** [review]
Nice!

File: docs/language/run-time-types.md (hunk 1 above)
Comment (SupunS) [reply docs/language/run-time-types.md]: Added with https://github.com/onflow/cadence/pull/737/commits/ebbaeabd38007509199b52086041f2fa2dc57371

**Pull request merged**